src/topp/IDFileConverter.cpp

```diff
@@ -170,6 +176,9 @@ class TOPPIDFileConverter :
     registerFlag_("ignore_proteins_per_peptide", "[Sequest only] Workaround to deal with .out files that contain e.g. \"+1\" in references column,\n"
                                                  "but do not list extra references in subsequent lines (try -debug 3 or 4)", true);
     registerStringOption_("scan_regex", "<expression>", "", "[Mascot, pepXML, Percolator only] Regular expression used to extract the scan number or retention time. See documentation for details.", false, true);
+    registerFlag_("add_spectrum_references", "Adds the spectrum files spectrum ID reference to each PeptideIdentification. Warning: works only with mz_file set", true);
+    registerDoubleOption_("add_ionmatch_annotation", "<tolerance value>", 0.5, "Adds IonMatch statistics to each PeptideHit if tolerance value > 0. Warning: works only with mz_file set, might be time and memory laborious!", false, true);
+    registerFlag_("reset_basename", "Resets the BaseName of every PeptideIdentification with the basename of the given mzML file.", true);
```

Comment:
hmm I am not sure if setting this  in the PI this is sufficient.
I would also call this spectra_data as in ProteinIdentification/Feature and ConsensusFeature.
In the file name list in ProteinIdentification it might need to be updated to.